packages/s/sway/files/50-systemd-user.conf

```diff
@@ -2,4 +2,15 @@
 # See Arch FS#63021
 # Adapted from xorg's 50-systemd-user.sh, which achieves a similar goal.
 
-exec systemctl --user import-environment DISPLAY WAYLAND_DISPLAY SWAYSOCK XDG_CURRENT_DESKTOP
+# Upstream refuses to set XDG_CURRENT_DESKTOP so we have to.
+exec systemctl --user set-environment XDG_CURRENT_DESKTOP=sway
+exec systemctl --user import-environment DISPLAY \
+                                         SWAYSOCK \
+                                         WAYLAND_DISPLAY \
+                                         XDG_CURRENT_DESKTOP
+
+exec hash dbus-update-activation-environment 2>/dev/null && \
```

Comment:
Would probably be kind to leave a message here explaining why the `hash` statement is necessary for the sake of future maintainers?

Reply:
TBH I'm not actually sure, I just copied that from the corresponding Arch package. We can look into it once this lands.